novuRajan/gannt-chart
# Custom classs in bar

## Description of the problem this PR solves
Ability to add custom class in a bar

## Testing instruction
Add custom_class property in task object
now inspect in the bar ur custom class will appear
```
 [
                {
                    id: 2,
                    name: 'Create a new Laravel project using Composer.',
                    start: '2023-10-08',
                    end: '2023-10-13',
                    progress: 50,
custom_class:"your-own-class"
                    dependencies: []
                },]
```
## Browsers tested
Select all browsers the solution was tested on <br>
*(test on as many as possible if the solution is front-end, otherwise this section is optional)*

## Notes
